### PR TITLE
docs: update bug report template

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
docs

**Did you add tests for your changes?**
No
**If relevant, did you update the documentation?**
yes
**Summary**

Many times developers do not write about the version of packages they are using and we have to ask them manually. Adding it to the issue template will save the developer's time.

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Review by alexander-akait — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Steps to reproduce the behavior:
 
 <!-- If applicable, add screenshots to help explain your problem. -->
 
-**Please paste the results of `webpack-cli info` here, and mention other relevant information**
+**Please paste the results of `webpack-cli info` and `webpack-cli version` here, and mention other relevant information**
```
Comment (alexander-akait) [inline .github/ISSUE_TEMPLATE/Bug_report.md:29]: Is it `webpack-cli info` is not output versions?

Comment (snitin315) [reply .github/ISSUE_TEMPLATE/Bug_report.md:29]: Actually, it does. Don't know why it didn't when I tested last time (maybe I wasn't in the correct repository while testing earlier)

Comment (alexander-akait) [reply .github/ISSUE_TEMPLATE/Bug_report.md:29]: yep, I think `webpack-cli info` is enough, ideally we should ask this output in all issue templates, better to use `npx --no-install webpack-cli info`

Comment (snitin315) [reply .github/ISSUE_TEMPLATE/Bug_report.md:29]: I will update.

Comment (snitin315) [inline .github/ISSUE_TEMPLATE/Bug_report.md:29]:
```suggestion
**Please paste the results of `npx --no-install webpack info` here, and mention other relevant information**
```

**Review by alexander-akait — APPROVED** [review]

**Review by evenstensberg — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Steps to reproduce the behavior:
 
 <!-- If applicable, add screenshots to help explain your problem. -->
 
-**Please paste the results of `webpack-cli info` here, and mention other relevant information**
+**Please paste the results of `npx --no-install webpack info` here, and mention other relevant information**
```
Comment (alexander-akait) [inline .github/ISSUE_TEMPLATE/Bug_report.md:29]: Let's `npx webpack info`, we should not use `--no-install` for info

**Review by alexander-akait — CHANGES_REQUESTED** [review]
Let's do it better :star:

**Review by alexander-akait — APPROVED** [review]

**Review by rishabh3112 — APPROVED** [review]

**Pull request merged**